src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -206,6 +206,11 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             ss << RS_(L"CopyTextCommandKey").c_str();
         }
 
+        if (WithControlSequences())
+        {
+            ss << L", withControlSequences: true";
+        }
+

```

Comment:
I wasn't sure if I should have actual, translated special case names pulled from the resources when this is set (eg `Copy with control sequences` and `Copy single line with control sequences`). Happy to make that change if it's preferred!